### PR TITLE
Allow bottom panel to be hidden

### LINKED ISSUE #589: Settings for hidding the list of errors at the bottom
Option to toggle this list :

image

Related : https://github.com/AtomLinter/Linter/issues/579 (specifically: https://github.com/AtomLinter/Linter/issues/579#issuecomment-114679665) but deserves its own issue for tracking.

### PR DESCRIPTION
**Pull request by steelbrain**
Just click the active tab to hide it
Click any tab to show it again
Fixes #589

ToDo
 - [x] Also add a setting for this

### CODE REVIEW

**Comment by k2b6s9j** [comment]
Looks great! :shipit:

**Comment by AsaAyers** [comment]
Looks great. It works and you even persisted whether it's hidden :+1: :+1:

**Comment by AsaAyers** [comment]
Changelog and then merge?

**Comment by steelbrain** [comment]
Okie dokie

**Comment by iam4x** [comment]
Perfect thank's @steelbrain !

File: styles/linter-plus.less [hunk 1]
```diff
@@ -15,6 +15,9 @@ linter-message {
   overflow-y: scroll;
   max-height: 150px;
   padding: 4px 8px;
+  &[hidden]{
```
Comment (ssorallen) [inline styles/linter-plus.less]: This is the native styling for the `hidden` attribute; you shouldn't need to add this.

Comment (steelbrain) [inline styles/linter-plus.less]: native styling of `[hidden]{display:none}` is overridden when we use a more direct selector and set `display:block`.

Comment (steelbrain) [inline styles/linter-plus.less]: `<div id="test" hidden></div>`
```css
#test{ display: block; }
```
Now that test div won't be hidden anymore despite having that hidden attribute, so we have to add override it again
```css
#test{ display: block; }
#test[hidden] { display: none  }
```

**Comment by ssorallen** [comment]
Aha got it, I took that out of context. Why is `display: block` needed on the `div` elements created here?

File: styles/linter-plus.less (hunk 1 above)
Comment (ssorallen) [inline styles/linter-plus.less]: Aha got it, I took that out of context. Why is display: block needed on the div elements created here?

Comment (steelbrain) [inline styles/linter-plus.less]: It was added in a PR.  https://github.com/AtomLinter/linter-plus/pull/17

Comment (ssorallen) [inline styles/linter-plus.less]: Strange, removing that `display: block` still works fine for me. Not worth picking it apart though, it's small and it works! Thanks for clarifying.

**Pull request merged**